modules/ui/raw_tag_editor.js

```diff
@@ -75,13 +81,71 @@ export function uiRawTagEditor(context) {
             rowData.push({ index: _indexedKeys.length, key: '', value: '' });
         }
 
-        // List of tags
+
+        // View Options
+        var options = wrap.selectAll('.raw-tag-options')
+            .data([0]);
+
+        var optionsEnter = options.enter()
+            .append('div')
+            .attr('class', 'raw-tag-options');
+
+        var optionEnter = optionsEnter.selectAll('.raw-tag-option')
+            .data(availableViews, function(d) { return d.id; })
+            .enter();
+
+        optionEnter
+            .append('button')
+            .attr('class', function(d) {
+                return 'raw-tag-option raw-tag-option-' + d.id + (_tagView === d.id ? ' selected' : '');
+            })
+            .attr('title', function(d) { return d.id; })
+            .on('click', function(d) {
+                _tagView = d.id;
+                context.storage('raw-tag-editor-view', d.id);
+
+                wrap.selectAll('.raw-tag-option')
+                    .classed('selected', function(datum) { return datum === d; });
+                wrap.selectAll('.tag-text')
+                    .classed('hide', (d.id !== 'text'));
+                wrap.selectAll('.tag-list, .add-row')
+                    .classed('hide', (d.id !== 'list'));
+            })
+            .each(function(d) {
+                d3_select(this)
+                    .call(svgIcon(d.icon));
+            });
+
+
+        // View as Text
+        var textarea = wrap.selectAll('.tag-text')
+            .data([0]);
+
+        textarea = textarea.enter()
+            .append('textarea')
+            .attr('class', 'tag-text' + (_tagView !== 'text' ? ' hide' : ''))
+            .call(utilNoAuto)
+            .merge(textarea);
+
+        textarea
+            .attr('rows', rowData.length + 1)
```

Comment:
Can we disable spell checking and autocorrect? Safari’s autocorrect is pretty aggressive inside textareas, and the 〰️ underlines get pretty annoying with raw tags.

Reply:
I dunno sometimes it does the right thing :trollface: 

<img width="374" alt="Screenshot 2019-05-03 10 09 51" src="https://user-images.githubusercontent.com/38784/57142881-9f2e3c80-6d8b-11e9-9f51-b1f8e61a4d49.png">